config/routes.rb

```diff
@@ -3,9 +3,16 @@
   resources :merchants do
 
     resources :invoices, controller: 'merchant_invoices'
-    
+
     resources :items, controller: 'merchant_items', except: [:destroy]
 
     resources :dashboard, controller: 'merchant_dashboard', only: [:index]
   end
+
+  resources :admin, only: [:index]
+
+  namespace :admin do
```

Comment:
interesting I wanna play around with this a bit when I pull to see what we can do with the other resources/controller syntax